### PR TITLE
Toolkit: Remove changelog command

### LINKED ISSUE #56028: Toolkit: Remove `changelog` command (already replaced by a github workflow)

### PR DESCRIPTION
**Pull request by gitstart**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #56028

# Release notice breaking change

Removing the unused `changelog` command in `@grafana/toolkit`.

### CODE REVIEW

**Comment by gitstart** [comment]
Hello @academo, I hope this is the expected solution for #56028. If not, happy to make any further required changes 😃

**Comment by academo** [comment]
> Hello @academo, I hope this is the expected solution for #56028. If not, happy to make any further required changes smiley

even though the command is removed we also need to remove all the associated files that contained the command runner task.

File: packages/grafana-toolkit/src/cli/index.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import chalk from 'chalk';
 import { program } from 'commander';
 
-import { changelogTask } from './tasks/changelog';
```
Comment (academo) [inline packages/grafana-toolkit/src/cli/index.ts:4]: We also need to remove all the code associated to the command, in this case the `changelogTask` here `'./tasks/changelog'` and if there are other files associated that are only used there, delete them too.

We also want to see if there are nodejs dependencies that are used only there and nowhere else and if so, remove them from the package.json of the project.

**Review by academo — CHANGES_REQUESTED** [review]
Thanks for your contribution!. Please also remove all the associated files that contained the command runner task and verify if we can remove any nodejs dependency that was only used in this command from the package.json

**Comment by academo** [comment]
Also there are some betterer lines to remove https://github.com/grafana/grafana/blob/main/.betterer.results#L1112

Please make sure to install and run the pre-commit hooks

**Comment by gitstart** [comment]
Thank you for the review @academo, will get back to you with required changes

**Comment by gitstart** [comment]
Hi @academo! PR was updated with the changes, kindly review again

**Review by academo — APPROVED** [review]
Thanks for the changes and your contribution.

**Pull request merged**